### DISCUSSION TITLE
Is data source sending grafana variables(scopedVars, timeRange...etc) to back end API.

### DISCUSSION THREAD

**Discussion by Bukka-Dinakar** [grafana/grafana-infinity-datasource#369] [Q&A] [answered]
Is infinity data source sending grafana variables(scopedVars, timeRange...etc) to back end API in query params or in header?

**Comment by yesoreyeram** [comment, marked as answer]
They are not sent to backend. They interpolated in the frontend and the updated query sent to backend via post body

**Reply by Bukka-Dinakar** [reply]
Updated query is sent to backend in which form???
 request body or query param or header?

**Reply by yesoreyeram** [reply]
you can inspect the browser developer tools network tab.
